tests/API/Management/ConnectionsTestMocked.php

```diff
@@ -38,6 +58,10 @@ public function testConnectionsGetAll()
         $this->assertEquals( 'GET', $api->getHistoryMethod() );
         $this->assertEquals( 'https://api.test.local/api/v2/connections', $api->getHistoryUrl() );
         $this->assertEmpty( $api->getHistoryQuery() );
+
+        $headers = $api->getHistoryHeaders();
+        $this->assertEquals( 'Bearer __api_token__', $headers['Authorization'][0] );
+        $this->assertEquals( self::$telemetry, $headers['Auth0-Client'][0] );
```

Comment:
If I got this correctly, the telemetry is being set internally but you have a "copy" (not a real copy) of those values to be sent in the setUp method just to compare. Right? That's fine 👌 . 
If instead this value from the setUp method is magically being set on the `connections->getAll()` call I'd require a clarification please 😛

Reply:
> the telemetry is being set internally but you have a "copy" (not a real copy) of those values to be sent in the setUp method just to compare. Right?

Correct. `self::$telemetry` is the expected value, not a reference or anything else. I'll change that to `self::$expectedTelemetry` in another PR.